server/views/pages/categoriserDone.html

```diff
@@ -20,7 +20,7 @@
   <h1 class="govuk-heading-l">Prisoner Categorisation</h1>
 </div>
 
-<div class="govuk-tabs" data-module="tabs">
```

Comment:
we dont need the javascript as we are loading a new page not hiding and showing - plus it was causing console errors